### PR TITLE
Migrate ODK to Gradle

### PR DESCRIPTION
**Pull request by dcluna**
Migrating ODK to Gradle.

Depends on #109.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        minSdkVersion 9
+        targetSdkVersion 21
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_7
+        targetCompatibility JavaVersion.VERSION_1_7
+    }
+
+    def sourceLocations = ['app/src'
```
Comment (ctsims) [inline build.gradle]: Shouldn't the location of ../commcare be parameterized somehow/somewhere?

File: gradlew.bat [hunk 2]
```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
```
Comment (ctsims) [inline gradlew.bat]: should these files be here?

File: settings.gradle [hunk 3]
```diff
@@ -0,0 +1,3 @@
+include ':libraries:mapballoons'
+include ':javarosa'
+project(':javarosa').projectDir = new File('../javarosa/core')
```
Comment (ctsims) [inline settings.gradle]: CommCare's project dir should also be set up like this, right?

File: gradlew.bat (hunk 2 above)
Comment (dcluna) [inline gradlew.bat]: Yes, it's the gradle wrapper that runs with Android Studio

File: settings.gradle (hunk 3 above)
Comment (dcluna) [inline settings.gradle]: Only if it has a build.gradle (currently it doesn't)

**Pull request merged**